projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -285,6 +287,13 @@ function NewsletterPostPublishSettingsPanel( {
 					</div>
 				</PluginPostPublishPanel>
 			) }
+
+			{ /*
+			     Only show the following panel on WPCOM sites since the
+			     Fees and "Upgrade plan" links displayed are WPCOM specific.
```

Comment:
Could we do something similar for Jetpack sites? afaik, the process is the same there, paid plans give you access to different commission fees?

Reply:
We could! I had a few concerns here that I've yet to look into:

1. Would the memberships/earnings endpoint return the correct commission for JP sites? 
2. What would be the upgrade URL for JP sites?